### PR TITLE
[DOCS] Links transforms in aggregation docs

### PR DESCRIPTION
**Pull request by szabosteve**
This PR adds a new section to the Misc aggregation docs called "Indexing aggregation results with transforms".

Preview:  http://elasticsearch_52563.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indexing-aggregation-results.html

### CODE REVIEW

**Review by hendrikmuhs — APPROVED** [review]
LGTM

File: docs/reference/aggregations/misc.asciidoc [hunk 1]
```diff
@@ -167,3 +167,12 @@ request. This is the case for Terms, Significant Terms and Percentiles aggregati
 also contains information about the type of the targeted field: `lterms` (for a terms aggregation on a Long field),
  `sigsterms` (for a significant terms aggregation on a String field), `tdigest_percentiles` (for a percentile
  aggregation based on the TDigest algorithm).
+ 
+
+[[indexing-aggregation-results]]
+== Indexing aggregation results with {transforms}
+ 
+{ref}/transforms.html[Transforms] enable you to convert existing {es} indices
```
Comment (lcawl) [inline docs/reference/aggregations/misc.asciidoc]: Not a mandatory change, but this link is within the same book:
```suggestion
<<transforms,{transforms-cap}>> enable you to convert existing {es} indices 
```

**Review by lcawl — APPROVED** [review]
One suggestion, otherwise LGTM

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: 1d89511
7.6: e433185

**Pull request merged**